core/domain/learner_progress_domain.py

```diff
@@ -60,6 +61,10 @@ def __init__(
                 explorations in the learner playlist.
             collection_playlist: list(CollectionSummary). The summaries of the
                 collections in the learner playlist.
+            all_topic_summaries: list(TopicSummary). The summaries of the topics
+                in the edit goals.
+            new_topic_summaries: list(TopicSummary). The summaries of the topics
+                not tracked for the user.
```

Comment:
Oh, can you rename this? `new_topic_summaries` is not descriptive at all. You can just name it `untracked_topic_summaries` or smtg like that.

Reply:
Done.